### PR TITLE
Dashboard: Tooltip for balance on a particular day

### LINKED ISSUE #5617: Show balance on a particular day in dashboard
Several times already, users have echoed feedback that it's not possible for them to check their earnings on a particular day. 

Screenshot 2023-12-26 at 00 49 35

There are two possible way to approach this, or both:

1. Currently we show a chart line but if you hover over dots you don't really see the balance on a day. So we could show balance on particular day there Screenshot 2023-12-26 at 00 44 13
2. Add ability to show balance on particular day from the calendar

We should discuss this first before implementing.

### PR DESCRIPTION
**Pull request by dennisreimann**
Closes #5617.

Peek 2024-01-08 19-16

### CODE REVIEW

**Review by Kukks — APPROVED** [review]
ACK, but I think we usually have a test that verifies the hash of the js files?

**Comment by dennisreimann** [comment]
That's problematic in this case, because afaik it isn't distributed and I had to do a modification to it, because the [source](https://github.com/ant7/chartist-plugin-tooltip2) is six years old.

**Review by matman7x — APPROVED** [review]

**Review by dstrukt — APPROVED** [review]
Debated if I thought we should also put the unit `{X} {USD/BTC/etc..}` on the tooltip, but think it's very clear given the context. 

tACK - nice update!

File: BTCPayServer/Components/StoreWalletBalance/Default.cshtml [hunk 1]
```diff
@@ -64,32 +64,40 @@
...
-                    labelInterpolationFnc: value => rate
-                        ? DashboardUtils.displayDefaultCurrency(value, rate, defaultCurrency, divisibility).toString()
-                        : value
+                    labelInterpolationFnc: valueTransform
                 }
             };
             
             const render = data => {
                 let { series, labels } = data;
+                console.log(series, labels)
```
Comment (NicolasDorier) [inline BTCPayServer/Components/StoreWalletBalance/Default.cshtml:80]: debug statement

**Pull request merged**